api/src/Entity/ContentNode.php

```diff
@@ -44,6 +45,7 @@
 )]
 #[ApiFilter(filterClass: SearchFilter::class, properties: ['contentType', 'root'])]
 #[ApiFilter(filterClass: ContentNodePeriodFilter::class)]
+#[ApiFilter(filterClass: ContentNodeCampFilter::class)]
```

Comment:
Why do you even need this filter? I see no usage of it.

If you do need it, do you really need it to also return content nodes of the periods? If not, you could instead write this as follows:
```suggestion
#[ApiFilter(filterClass: SearchFilter::class, properties: ['contentType', 'root', 'root.activity.camp'])]
#[ApiFilter(filterClass: ContentNodePeriodFilter::class)]
```
Then we don't need to maintain a custom `ContentNodeCampFilter`.